src/Boscia.jl

```diff
@@ -3,6 +3,8 @@ module Boscia
 using FrankWolfe
 import FrankWolfe: compute_extreme_point
 export compute_extreme_point
+import Base: copy
+export copy 
```

Comment:
why do we need to reexport copy?

Reply:
> for example, we do not actually need to copy the solver right?

yes, in that case you can have the `copy` operation just return the same structure

Reply:
I removed it. We only need this for MOI supported solvers and those have their own strong branching function.